arch/x86/kvm/vmx/pkvm_init.c

```diff
@@ -448,6 +469,12 @@ static bool mitigate_bhi(void)
  * 6) X86_BUG_BHI:
  * Set SPEC_CTRL_BHI_DIS_S in spec ctrl MSR to mitigate BHI bug.
  *
+ * 7) X86_BUG_EIBRS_PBRSB.
+ * Fill RSB after vmexit.
```

Comment:
The comment above still says "2.4) PBRSB-eIBRS not affected on PTL, no need to fill RSB for vmexits." - it should be removed now?

Also the following should be updated now (both in this comment and in the comment before the `possible_cpu_bugs` definition)?

s/based on Intel PTL CPU/based on Intel PTL and ADL CPUs/
s/beyond those CPUs/
s/older than PTL/older than ADL/